src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,30 @@
-export const TodoInfo = () => {};
+import React from 'react';
+import { userInfo } from '../UserInfo';
+
+type TodoInfoProps = {
+  id: number;
+  title: string;
+  completed: boolean;
+  userId: number;
+  userEmail: string;
+  userName: string;
+};
+
+export const TodoInfo: React.FC<TodoInfoProps> = ({
+  id,
+  title,
+  completed,
+  userEmail,
+  userName,
+}) => {
+  return (
+    <article
+      data-id={id}
+      className={`TodoInfo ${completed ? 'TodoInfo--completed' : ''}`}
+    >
+      <h2 className="TodoInfo__title">{title}</h2>
+      <userInfo email={userEmail}
```

Comment:
```suggestion
      <UserInfo email={userEmail}
```
should be 
<img width="601" alt="image" src="https://github.com/user-attachments/assets/fa709815-5ba5-4011-9ae2-e3742e78dd55">

Reply:
and don't forget to close the component